06-building-a-program-start-to-finish/writing-tests/output/invertedIndexSpec.js

```diff
@@ -0,0 +1,263 @@
+const books = require('../assets/books.json');
+const wrongData = require('../assets/wrongData.json');
+const emptyFile = require('../assets/emptyFile.json');
+const smallFile = require('../assets/smallFile.json');
+
+describe('InvertedIndex Class', () => {
+  beforeAll(() => {
+    this.invertedIndex = new InvertedIndex();
+  });
+
+  describe('Class instantaion', () => {
+    it('should be instantiated with the "new" keyword', () => {
+      const init = () => {
+        InvertedIndex();
+      };
+      expect(init)
+        .toThrowError(`Class constructor` +
```

Comment:
We have talked about this and I believe you can find a better way of handling this. Have you thought of putting this string in a file and import it so that you can make us of it on this line.